jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ThreadLimitHandler.java

```diff
@@ -204,7 +208,7 @@ private Remote getRemote(Request baseRequest)
         Remote remote = _remotes.get(ip);
         if (remote == null)
         {
-            Remote r = new Remote(ip, limit);
+            Remote r = new Remote(baseRequest.getContext(), ip, limit);
```

Comment:
[I could only add a comment to modified code sections, so the comment below refers to other parts of code].

Since it's almost 2023, can we make the parameterless constructor default to `true` for the rfc7239 parameter?

Also, method `getAndClose()` is unused and should be removed.